doc/refman/RefMan-tac.tex

```diff
@@ -1284,6 +1308,13 @@ \subsection{\tt assert ( {\ident} :\ {\form} )}
 
   \ErrMsg \errindex{Variable {\ident} is already declared}
 
+\item \texttt{eassert {\form} as {\intropattern} by {\tac}}\tacindex{eassert}\tacindex{eassert as}\tacindex{eassert by}\\
+  {\tt assert ( {\ident} := {\term} )}
+
+  While the different variants of \texttt{assert} expect that no
+  existential variables are generated by the tactic, \texttt{eassert}
+  removes this constraint.
```

Comment:
I would add: this allows not to specify the asserted statement completely before starting to prove it.
To me, this is the most useful feature provided by this PR. It should be described more positively.